bin/bwallet-cli

```diff
@@ -347,6 +347,14 @@ class CLI {
     this.log('Zapped!');
   }
 
+  async abandonTX() {
+    const hash = this.config.str(0);
+
+    await this.wallet.abandon(hash);
+
+    this.log('Abandoned!');
```

Comment:
Nit: I think it should log the hash here as well

Reply:
I don't see a similar log to match the format for this.

Reply:
I guess: "Abandoned tx: <hash>" ?

Reply:
As it's just the argument passed in, it may not be necessary, similar with other similar cases.